dbx.py

```diff
@@ -0,0 +1,125 @@
+import os
+from pyspark.dbutils import DBUtils
+
+MANIFEST_FILE_EXT = 'file_list'
+
+def export_parquet(parquet_path, spark):
+    dbutils = DBUtils(spark)
+    file_list = deep_ls(dbutils, parquet_path, 10)
+    manifest = ''
+    for file in file_list:
+        manifest += file.path + '\n'
```

Comment:
this is is some kind of "standardized" manifest?

Reply:
it's a file path per line

Reply:
It's not standard, it is used to reconstruct the folder structure on the other end

Reply:
why not use a JSON then?

Reply:
good question, never crossed my mind, didn't need any structure

Reply:
mainly because that "\n" raises a flag of feature problems :) + it's not extensible which might come in useful if it's to reconstruct something

Reply:
yeah, I hope it's not permanent, it's just a workaround until we export the structure, then none of this will be necessary.